src/content/testing/getting-started.mdx

```diff
@@ -0,0 +1,288 @@
+  - https://movies-cindy.okteto.example.com
+ ✓  Development environment 'getting-started' successfully deployed
+ ```
+
+## Step 2: Execute the `frontend` unit tests in a Test Container
+
+The [test](reference/okteto-manifest.mdx#dev-object-optional) section in the Okteto Manifest defines how to test the Movies App.
+
+For example, add the Test Container to run unit tests for the `frontend` service is like this:
+
+```yaml title="okteto.yaml"
+test:
+  frontend:
+    image: node:20
+    context: frontend
+    caches:
+      - yarn/.cache
+      - node_modules
+    commands:
+      - yarn install
+      - yarn test
+```
+
+`frontend` is the name of the Test Container. The meaning of the rest of fields is:
+
+- `image`: the image used as the runtime of your Test Container.
+- `context`: the folder that will be synchronized between your local machine and the Test Container.
+- `caches`: a list of folders to persist between executions of the Test Container. This is very important for performance of your Test Containers.
+- `commands`: a list of commands to execute your tests.
+
+Also, note that there is a `.oktetoignore` file in the `frontend` folder to indicate which files shouldn't be synchronized to your Test Container.
+This is useful to avoid synchronizing binaries, build artifacts, cache metadata, or dependencies like the `node_modules` folder.
+
+Next, execute the following command to run the `frontend` unit tests in a Test Container:
+
+```bash
+okteto test frontend
+```
+
+The first executon takes longer because the `caches` aren't initialize, but you will get this output:
+
+```bash
+ i  Using cindy @ okteto.example.com as context
+ i  Executing test container 'frontend'
+ i  Running 'yarn install'
+yarn install v1.22.22
+[1/4] Resolving packages...
+[2/4] Fetching packages...
+[3/4] Linking dependencies...
+[4/4] Building fresh packages...
+Done in 43.31s.
+ ✓  Command 'yarn install' successfully executed
+ i  Running 'yarn test'
+yarn run v1.22.22
+$ jest
+PASS src/components/Hero.test.jsx
+PASS src/components/TitleList.test.jsx
+PASS src/components/Item.test.jsx
+PASS src/components/App.test.jsx
+PASS src/components/UserProfile.test.jsx
+PASS src/components/ListToggle.test.jsx
+Test Suites: 6 passed, 6 total
+Tests:       20 passed, 20 total
+Snapshots:   0 total
+Time:        5.658 s
+Ran all test suites.
+Done in 6.59s.
+ ✓  Command 'yarn test' successfully executed
+ ✓  Test container 'frontend' passed 
+```
+
+If you rerun your Test Container:
+
+```bash
+okteto test frontend
+```
+
+```bash
+ i  Using cindy @ okteto.example.com as context
+ i  Executing test container 'frontend'
+ i  Skipping test container 'frontend', CACHED
+ ✓  Test container 'frontend' passed
+```
+
+You can see that Okteto skips the execution of the tests. This is because the content of the folder `context: frontend` didn't change.
+This is very convinient in monorepos, since Okteto won't rerun unit tests in the folder has no changes.
```

Comment:
```suggestion
This is very convenient in monorepos, since Okteto won't rerun unit tests in the folder has no changes.
```

Reply:
Or maybe a better phrasing, something like:
```suggestion
This is especially convenient in monorepos because Okteto will only rerun unit tests in folders with changes, saving time and resources.
```